### DISCUSSION TITLE
请问在 option:after_check 函数中，如何调用自定义的全局函数

### DISCUSSION THREAD

**Discussion by Leopard-C** [xmake-io/xmake#5445] [Q&A]
![image](https://github.com/user-attachments/assets/033ec99c-887a-4750-a6cc-bbb44708f7b7)

**Comment by waruqi** [comment]
不支持，故意限制的，描述域和脚本域 不要混着调。。

既然在脚本域，就走 import 去调用 脚本模块的 function

**Reply by Leopard-C** [reply]
知道怎么写了，谢谢！
